src/style.css

```diff
@@ -0,0 +1,9 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+@import "./allStyles/variables.css";
+@import "./allStyles/start--values.css";
+@import "./allStyles/image.css";
+@import "./allStyles/title.css";
+@import "./allStyles/code--name.css";
+@import "./allStyles/rating.css";
+@import "./allStyles/price.css";
+@import "./allStyles/button.css";
```

Comment:
There should be only 2 BEM blocks in your code card and stars all the others are just card elements.